lib/archethic/p2p/geo_patch.ex

```diff
@@ -93,4 +93,127 @@ defmodule ArchEthic.P2P.GeoPatch do
       x2 / 2
     end
   end
+
+  @doc """
+  Get range of longitude / latitude coordinates from geo patch
+  """
+  @spec from_geo_patch(binary()) :: list(list(float()))
```

Comment:
Firstly, thanks for the contribution
But I would suggest renaming it `to_coordinates`, as it will bring more meaning to the function without reading it.
Also, the returning list can be a tuple I think, as it will be a fixed size.

Reply:
Updated in https://github.com/archethic-foundation/archethic-node/pull/238/commits/5054594bb22ca132aa8dcbdab05aac81a2adcb85